tests/test_chi_ssa_18.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import sys
+import pytest
+from chi_ssa_18 import ChiSsa18Spider
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+sys.path.append('city-scrapers\city_scrapers\spiders')
```

Comment:
We shouldn't modify the path here. The line importing the spider can be the following to fix this:

```python
from city_scrapers.spiders.chi_ssa_18 import ChiSsa18Spider
```